cdap-explore/src/test/java/co/cask/cdap/explore/service/BaseHiveExploreServiceTest.java

```diff
@@ -99,10 +99,15 @@
     .registerTypeAdapter(Schema.class, new SchemaTypeAdapter())
     .create();
 
-  // TODO: When datasets are namespaced, we should add tests for multiple, non-default namespaces.
   protected static final Id.Namespace NAMESPACE_ID = Id.Namespace.from("default");
+  protected static final Id.Namespace OTHER_NAMESPACE_ID = Id.Namespace.from("other");
```

Comment:
eventually, we should probably get to a state where we only use default namespace in tests that:
1. test v2 apis
2. test backward compatibility